Formula/bnd.rb

```diff
@@ -0,0 +1,21 @@
+class Bnd < Formula
+  desc "the Swiss Army Knife for OSGi bundles"
+  homepage "http://bnd.bndtools.org/"
+  url "https://search.maven.org/remotecontent?filepath=biz/aQute/bnd/biz.aQute.bnd/3.3.0/biz.aQute.bnd-3.3.0.jar"
+  sha256 "b6b68dfcd0f5ba767a202bf35eb3eb964c63e679e8217dd514dac807e6cedee8"
+
+  bottle :unneeded
+
+  def install
+    prefix.install "biz.aQute.bnd-#{version}.jar"
+
+    (bin/"bnd").write <<-EOS.undent
+      #!/bin/bash
+      exec java -jar #{prefix}/biz.aQute.bnd-#{version}.jar "$@"
+    EOS
+  end
+
+  test do
+    assert_match /^#{version}/, shell_output("#{bin}/bnd version")
```

Comment:
This is a good start, but something more complex than a simple `version`/`help` test is usually preferred. Is that possible with `bnd`?